test/test_parsers.py

```diff
@@ -47,4 +48,35 @@ def test_zenodo_parser(self):
 
 
 
+class TestArxiv(unittest.TestCase):
```

Comment:
I suggest adding test cases for arXiv records where the first author's initial is a non-ascii character.